public/app/features/dashboard-scene/settings/VersionsEditView.tsx

```diff
@@ -1,15 +1,23 @@
+import { css } from '@emotion/css';
 import React from 'react';
 
-import { PageLayoutType, dateTimeFormat, dateTimeFormatTimeAgo } from '@grafana/data';
+import { GrafanaTheme2, PageLayoutType, dateTimeFormat, dateTimeFormatTimeAgo } from '@grafana/data';
 import { SceneComponentProps, SceneObjectBase, sceneGraph } from '@grafana/scenes';
-import { HorizontalGroup, Spinner } from '@grafana/ui';
+import { HorizontalGroup, Spinner, useStyles2 } from '@grafana/ui';
 import { Page } from 'app/core/components/Page/Page';
 
 import { DashboardScene } from '../scene/DashboardScene';
 import { getDashboardSceneFor } from '../utils/utils';
 
 import { DashboardEditView, DashboardEditViewState, useDashboardEditPageNav } from './utils';
-import { RevisionsModel, VersionHistoryTable, historySrv } from './version-history';
+import {
+  RevisionsModel,
+  VersionHistoryComparison,
+  VersionHistoryHeader,
+  VersionHistoryTable,
+  VersionsHistoryButtons,
+  historySrv,
```

Comment:
@mdvictor - I see these being duplicated in the old runtime directory and here. Think we should only keep the version in `dashboard-scene` dir, and get rid of the old runtime versions of these files. 

My point was - let's move what's possible to move without a major refactor under the `dashboard-scene` feature dir, and reference those in the old architecture imports.

Reply:
I migrated these files and left the others in the old `VersionHistory` folder for now, they will be touched in the restoration PR.